File: tourneyAPI/Models/Entities/Characters/Falco.cs
namespace Entities;

using Enums;

// Defines the competitive profile metadata for this playable character.
public class Falco : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public Falco()
    {
        Id = CharacterId.Falco;
        characterName = CharacterName.FALCO;
        archetype = Archetype.MIX_UP;
        fallSpeed = FallSpeed.FAST_FALLERS;
        weightClass = WeightClass.FEATHERWEIGHT;
        tierPlacement = TierPlacement.A;
    }

}

File: tourneyClient/src/models/entities/Characters/Falco.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { CharacterName } from "@/models/Enums/CharacterName";
import { Archetype } from "@/models/Enums/Archetype";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const Falco: Character =
{
    id: CharacterId.Falco,
    characterName: CharacterName.FALCO,
    archetype: Archetype.MIX_UP,
    fallSpeed: FallSpeed.FAST_FALLERS,
    weightClass: WeightClass.FEATHERWEIGHT,
    tierPlacement: TierPlacement.A
}

export default Falco; 